.pre-commit-config.yaml

```diff
@@ -15,14 +15,29 @@
 # limitations under the License.
 #
 repos:
+  - repo: https://github.com/MarcoGorelli/auto-walrus
+    rev: v0.2.2
+    hooks:
+      - id: auto-walrus
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.4.0
+    hooks:
+      - id: pyupgrade
+        args:
+          - --keep-runtime-typing
+          - --py38-plus
+  - repo: https://github.com/hadialqattan/pycln
+    rev: v2.1.2
+    hooks:
+      - id: pycln
+        args:
+          - --disable-all-dunder-policy
+          - --exclude=superset/config.py
```

Comment:
`pycln` chokes on `module not found` errors when we try importing the default configs.